### PR TITLE
ADL Graph Pattern

### PR DESCRIPTION
**Pull request by PormpimonPang**
Find and try to show you the best meaningful ADL graph patterns.

### CODE REVIEW

**Comment by PormpimonPang** [comment]
Now it's done but wait for the advisor to agree again next meeting and delete unused before merge.

File: lib/widget/radial_gauge_chart .dart [hunk 1]
```diff
@@ -4,95 +4,95 @@ import 'package:syncfusion_flutter_gauges/gauges.dart';
...
+                    //   Colors.red,
+                    //   Colors.orange,
+                    //   Colors.yellow,
+                    //   Colors.green
+                    // ], stops: <double>[
+                    //   0.10,
+                    //   0.25,
+                    //   0.50,
+                    //   1
+                    // ]),
```
Comment (Thaweesak-sai) [inline lib/widget/radial_gauge_chart .dart:29]: If this is not needed anymore. remove it. else use //TODO to remind things

File: lib/widget/test_chart.dart [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                                            fontSize: 16))),
+                                angle: 90,
+                                positionFactor: 0.2)
+                          ]),
+                    ],
+                  ),
+                )
+              ],
+            ),
+            //for (var i in [1]) buildItem("Item ${i}"),
```
Comment (Thaweesak-sai) [inline lib/widget/test_chart.dart:151]: this as well

**Review by Thaweesak-sai — COMMENTED** [review]
Leave some comments

File: lib/widget/radial_gauge_chart .dart (hunk 1 above)
Comment (PormpimonPang) [reply lib/widget/radial_gauge_chart .dart:29]: I wait for a consult today and change to the final dashboard pattern

File: lib/widget/test_chart.dart (hunk 2 above)
Comment (PormpimonPang) [reply lib/widget/test_chart.dart:151]: I'm waiting for today's consultation as well.

**Review by Thaweesak-sai — APPROVED** [review]
If it is good. then go

**Pull request merged**